<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Monitor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCustomPlot" name="customPlot_9" native="true"/>
     </item>
     <item>
      <widget class="QCustomPlot" name="customPlot_10" native="true"/>
     </item>
     <item>
      <widget class="QCustomPlot" name="customPlot_11" native="true"/>
     </item>
     <item>
      <widget class="QCustomPlot" name="customPlot_12" native="true"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCustomPlot" name="customPlot_5" native="true"/>
     </item>
     <item>
      <widget class="QCustomPlot" name="customPlot_6" native="true"/>
     </item>
     <item>
      <widget class="QCustomPlot" name="customPlot_7" native="true"/>
     </item>
     <item>
      <widget class="QCustomPlot" name="customPlot_8" native="true"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCustomPlot" name="customPlot" native="true"/>
     </item>
     <item>
      <widget class="QCustomPlot" name="customPlot_2" native="true"/>
     </item>
     <item>
      <widget class="QCustomPlot" name="customPlot_3" native="true"/>
     </item>
     <item>
      <widget class="QCustomPlot" name="customPlot_4" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
